src/lib/Components/Consignments/Components/DoneButton.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+
+import BottomAlignedButton from "./BottomAlignedButton"
+
+export interface DoneButtonProps {
+  onPress: () => void
+  children?: any[]
+}
+
+const render = (props: DoneButtonProps) => {
+  const doneButtonStyles = {
+    backgroundColor: "black",
+    marginBottom: 0,
+    paddingTop: 18,
+    height: 56,
+  }
+  return (
+    <BottomAlignedButton onPress={props.onPress} bodyStyle={doneButtonStyles} buttonText="DONE">
+      {props.children}
+    </BottomAlignedButton>
+  )
+}
+
+export default class DoneButton extends React.Component<DoneButtonProps, null> {
+  render() {
+    return render(this.props)
+  }
+}
```

Comment:
in case you didn't know - we do this extra Class at the end so [that HMR works](https://facebook.github.io/react-native/blog/2016/03/24/introducing-hot-reloading.html)